arm-advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,641 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "2016-07-12-preview",
+		"title": "AdvisorManagementClient",
+		"description": "REST APIs for Azure Advisor"
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"consumes": ["application/json"],
+	"produces": ["application/json"],
+	"paths": {
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/generateRecommendations": {
+			"post": {
+				"tags": ["GenerateRecommendations"],
+				"description": "Initiates the recommendation generation/computation process for a subscription. This is an asynchronous operation. The generated recommendations are stored in a cache in the Advisor service.",
+				"operationId": "Recommentations_Generate",
+				"parameters": [{
+					"$ref": "#/parameters/subscriptionIdParameter"
+				},
+				{
+					"$ref": "#/parameters/apiVersionParameter"
+				}],
+				"responses": {
+					"202": {
```

Comment:
Seems like this is a long-running operation. Reference: https://github.com/Microsoft/api-guidelines/blob/master/Guidelines.md#13-long-running-operations

What is the terminal status code for this endpoint. `202` seems confusing. Should be designed as long running that can follow location header to facilitate the ARM guidelines. 

**cc:**
@ravbhatnagar